base/kustomization.yaml

```diff
@@ -1,11 +1,5 @@
 resources:
-- clouddriver/deployment.yml
-- clouddriver/service.yml
-secretGenerator:
-- files: []
-  name: clouddriver-config
-- files: []
-  name: spinnaker-secrets
+- ./clouddriver
```

Comment:
This is really cool and I don't think I knew this until reading this PR...so a `kustomization.yaml` can either list resources explicitly or refer to directories which themselves contain `kustomization.yaml` and so on recursively?

Reply:
Yes! I think once we have more of these we can figure out the best way to organize these so it's not too confusing, but for now this seemed like a reasonable start.